### ISSUE TITLE
Error pulling with GitHub application whilst brackets is open

### ISSUE THREAD

**Issue by PhyscoKillerMonkey** [zaggino/brackets-git#972]
Have brackets open and sync a change from the GitHub using the GitHub for Windows application.
Swap back to brackets.io and brackets.io throws errors (one for every commit pulled) which fill up the entire screen.
The errors don't actually affect the files as far as I can see so possibly have errors in the console rather than onscreen?

--------------------------------

*Brackets 1.2.0-15697 (win), Git 1.9.5.msysgit.1 & Brackets-Git 0.14.18*

```
Error: Failed parsing branch name from a15752d2f103fbe17061fc9c31727a001a8a8821
```

```
Error: Failed parsing branch name from a15752d2f103fbe17061fc9c31727a001a8a8821
    at file:///C:/Users/Reece/AppData/Roaming/Brackets/extensions/user/zaggino.brackets-git/src/Branch.js:386:40
    at File.eval (/filesystem/File.js:125:13)
    at eval (/filesystem/impls/appshell/AppshellFileSystem.js:378:25)
```

**Comment by zaggino** [comment]
This should be fixed in latest version `0.14.19`, please test/confirm. Thanks!

**Comment by PhyscoKillerMonkey** [comment]
Seems to be fixed, thankyou :)
